lib/classes/Service.js

```diff
@@ -41,19 +41,33 @@ class Service {
       return BbPromise.resolve();
     }
 
+    let serverlessYamlPath = path.join(servicePath, 'serverless.yml');
```

Comment:
Good idea. I did this basically because all internal variables still refer to the `fooYaml` name (with yaml written out). It's e.g. used everywhere in the `Service` class. I kept it that way to stay consistent.